pkg/apply/processor/scale.go

```diff
@@ -233,7 +234,20 @@ func (c *ScaleProcessor) MountRootfs(cluster *v2.Cluster) error {
 		return err
 	}
 
-	return fs.MountRootfs(cluster, hosts, true, false)
+	return fs.MountRootfs(cluster, hosts)
+}
+
+func (c *ScaleProcessor) Bootstrap(cluster *v2.Cluster) error {
+	logger.Info("Executing pipeline Bootstrap in CreateProcessor")
+	hosts := append(c.MastersToJoin, c.NodesToJoin...)
```

Comment:
You may need to consider the logic of registry here.

Reply:
scale process currently don't support assign registry role. so nothing 
 really happened here.